plugins/source/test/resources/services/test.go

```diff
@@ -11,27 +11,29 @@ func TestSomeTable() *schema.Table {
 	return &schema.Table{
 		Name:        "test_some_table",
 		Description: "Test description",
-		Resolver:    fetchSomeTableData,
+		Resolver:    resolveSomeTable,
```

Comment:
Could we rename this back just for the sake of the less diff?

Reply:
done in [`714d1e9` (#11099)](https://github.com/cloudquery/cloudquery/pull/11099/commits/714d1e98df85fc04d99fb96f47c0032ce1cbc063)